lib/smart_proxy_pulp_plugin/pulpcore_plugin.rb

```diff
@@ -14,7 +14,7 @@ class PulpcorePlugin < ::Proxy::Plugin
     expose_setting :pulp_url
     expose_setting :mirror
     expose_setting :content_app_url
-    capability( lambda do ||
+     capability lambda { PulpcoreClient.capabilities }
```

Comment:
So the Github rendering was messing with me here

Reply:
which old code did you mean?

Reply:
Previously it was a multi line lambda. The GH suggestion only replaced the first line, keeping the second and third line. That broke the build. It's fine (and probably better) to keep this PR minimal and clean up the lambda later.